src/grammar.lua

```diff
@@ -453,3 +456,17 @@ function pmatch(patt)
     end
   end)
 end
+
+_M.anyOf	= anyOf
```

Comment:
We are trying to discourage the use of `_M` in modules. This is a remain from the early days of Lua modules; some influential users used `_M` back then and many people adopted it.

 The Lua manual explicitly says in http://www.lua.org/manual/5.3/manual.html#3.1

> As a convention, programs should avoid creating names that start with an underscore followed by one or more uppercase letters.

The new convention is to use the module name as its table name.